static/src/javascripts/projects/common/modules/identity/sign-in-gate/helper.js

```diff
@@ -49,6 +49,17 @@ export const hasUserDismissedGate: ({
     return !!prefs[`${name ? `${name}-` : ''}${variant}`];
 };
 
+// Dynamically sets the gate custom parameter for Google ad request page targeting
+export const setGatePageTargeting = (canShow: boolean): void => {
+    if (window && window.googletag) {
```

Comment:
`window` should always exist, so i think you just need to check for `window.googletag`

Reply:
fixed